### PR TITLE
Clarified Load by RDF page.

### LINKED ISSUE #1182: Clarify Load RDF tab
**Describe the bug**
When I go "Load RDF" and I enter the following data:

```
<http://example.com/foo> <http://id.loc.gov/ontologies/bibframe/mainTitle> "foo"@en .
<http://example.com/foo> <http://sinopia.io/vocabulary/hasResourceTemplate> "profile:bf2:Title:AbbrTitle" .
<http://example.com/foo> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://id.loc.gov/ontologies/bibframe/AbbreviatedTitle> .
```
and I set `http://example.com/foo` as the root.

Then when I click "Submit" and preview rdf, the root subject (http://example.com/foo) has been lost.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #1182

### CODE REVIEW

**Comment by justinlittman** [comment]
Screen Shot 2019-09-30 at 2 39 17 PM

**Comment by jgreben** [comment]
The clarification text looks good to me. But If the browser window is too small there is no way to see the entire Base URI example text.

Screen Shot 2019-09-30 at 3 08 16 PM

**Comment by jgreben** [comment]
Also, FWIW, when I try to submit the form nothing happens and I get a console error:

`TypeError: this.match(...).toArray(...)[0] is undefined`

Maybe that needs to become a new issue?

**Comment by justinlittman** [comment]
@jgreben Google didn't know how to make it so that the entire Base URI example can show up even if the browser window is made smaller. Do you?

**Comment by justinlittman** [comment]
@jgreben Regarding the `TypeError`: Can you try to replicate on https://github.com/LD4P/sinopia_editor/pull/1371? This PR includes improved error handling for this form.

**Comment by jgreben** [comment]
> @jgreben Google didn't know how to make it so that the entire Base URI example can show up even if the browser window is made smaller. Do you?

How about just making it a `<textarea>` instead?

**Comment by justinlittman** [comment]
I didn't want to make it a text area, as it might confuse the user into thinking that the input should be more than one line. Split into placeholder and help block instead.
Screen Shot 2019-10-01 at 5 17 20 AM

**Comment by jcoyne** [comment]
I agree that `<textarea>` is not an appropriate widget for a URI. I think this change is good.

File: src/components/load/LoadByRDFForm.jsx [hunk 1]
```diff
@@ -40,19 +40,31 @@ const LoadByRDFForm = (props) => {
...
         <div className="form-group">
           <label htmlFor="resourceTextArea">N3 RDF</label>
-          <textarea className="form-control" id="resourceTextArea" rows="15" value={resourceN3} onChange={handleResourceN3Change}></textarea>
+          <textarea className="form-control" id="resourceTextArea" rows="15" value={resourceN3}
+                    onChange={handleResourceN3Change} placeholder={n3PlaceHolder}></textarea>
         </div>
         <div className="form-group">
           <label htmlFor="uriInput">Base URI</label>
-          <input type="text" className="form-control" id="uriInput" value={baseUri} onChange={handleBaseUriChange} />
+          <input type="text" className="form-control" id="uriInput" value={baseUri} onChange={handleBaseUriChange}
```
Comment (jcoyne) [inline src/components/load/LoadByRDFForm.jsx]: `type` should be `url` See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/url

Comment (justinlittman) [reply src/components/load/LoadByRDFForm.jsx]: This is why I love code review. Always learning something new. (Or reminded of something I once knew.)

Changed.

**Pull request merged**